### PR TITLE
Example

### PR DESCRIPTION
**Pull request by carolynryan20**
what it does

### CODE REVIEW

File: readCSV.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 #    to read in the CSV and transform it into a better format
 # If no commandLine arguments are passed into it, then it passed the
 #    CSV into pandas.
+# This is an example comment to be deletedg
 def main():
```
Comment (carolynryan20) [inline readCSV.py]: This is an example :)

File: readCSV.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 #    to read in the CSV and transform it into a better format
 # If no commandLine arguments are passed into it, then it passed the
 #    CSV into pandas.
+# This is an example comment to be deletedg
```
Comment (govekk) [inline readCSV.py]: Nice comment

**Comment by bisilaj** [comment]
This is also a nice comment.

**Review by govekk — APPROVED** [review]

File: readCSV.py (hunk 2 above)
Comment (zephyrLucas) [reply readCSV.py]: Nice comment on a comment

**Review by bisilaj — DISMISSED** [review]
Great job! :)

**Review by zephyrLucas — APPROVED** [review]

**Review by zephyrLucas — DISMISSED** [review]
I'm not so sure if these shenanigans should be approved

**Review by bisilaj — APPROVED** [review]
This looks good.

**Review by zephyrLucas — CHANGES_REQUESTED** [review]
I still am not sure of these shenanigans

**Pull request closed without merging**